pkg/antctl/command_definition_test.go

```diff
@@ -108,13 +108,13 @@ func TestCommandDefinitionGenerateExample(t *testing.T) {
 	for k, tc := range map[string]struct {
 		use          string
 		cmdChain     string
-		singleObject bool
+		outputObject OutputType
 		expect       string
 	}{
-		"SingleObject": {
+		"outputObject": {
```

Comment:
This case is still testing single object, why change the case name?

Reply:
Updated. Thanks for reminding me cuz I changed it by mistake.